resources/i18n/en.json

```diff
@@ -1686,6 +1686,11 @@
     "SessionsDesc": "The number of compute sessions created.",
     "UsageHistory": "Usage History",
     "UsageHistoryNote": "The usage history shown here are based on terminated compute sessions.",
+    "description": {
+      "CPUUtilization": "Represents the amount of CPU time used by the session during the selected period. \n\nA single CPU core corresponds to a maximum of 1000%.\nIf multiple cores are allocated, the total usage can exceed 1000%, as it is calculated by summing up the maximum usage (1000%) of each core.",
+      "NetworkRx": "Indicates the network transfer rate.  \n\nRepresents the rate at which the container is receiving network data.",
+      "NetworkTx": "Indicates the network transfer rate. \n\n Represents the rate at which the container is sending network data."
```

Comment:
The part about "1000%" feels out of place.
Our capacity is in a situation of being 1000, and if we are to express it as a % to make it easy for the user to understand, I think we should show 1 core as 100%.

Reply:
I modified it to show it as a percentage. However, I know that the data on the prometheus side is designed to show the data as it comes down from the server (e.g., display_name will be added to response data in the future), but if this part needs to be shown as a percentage, it would be better to show it as a percentage on the server.